windows/src/test/verify_signatures/Makefile

```diff
@@ -0,0 +1,21 @@
+#
+# test for signatures and version information being correct in bin folder
+#
+
+!include ..\..\Defines.mak
+
+test: prereq
+  $(SIGCHECK) $(ROOT)\bin\desktop\* > sig1
+  $(SIGCHECK) $(ROOT)\bin\engine\* >> sig1
+  $(SIGCHECK) $(ROOT)\bin\inst\* >> sig1
+  $(VERIFY_SIGNATURES) < sig1
+
+# prereq may not be needed?
```

Comment:
Why this comment?  I think the prereq is needed to build the verify_signatures binary

Reply:
The prereq may already have been built, but it's not a big deal because it builds quickly anyway.